pkg/controller/networkpolicy/validate.go

```diff
@@ -1046,6 +1058,26 @@ func validateAntreaGroupSpec(s crdv1beta1.GroupSpec) (string, bool) {
 			return reason, allowed
 		}
 	}
+	return validateGroupIPBlocks(s.IPBlocks)
+}
+
+func validateGroupIPBlocks(ipbs []crdv1beta1.IPBlock) (string, bool) {
+	unicast, multicast := false, false
+	for i, ipb := range ipbs {
+		if reason, ok := validateIPBlock(&ipbs[i]); !ok {
+			return reason, ok
+		}
+		// CIDR formats are already validated in validateIPBlock()
+		ipaddr, _, _ := net.ParseCIDR(ipb.CIDR)
+		if ipaddr.IsMulticast() {
+			multicast = true
+		} else {
+			unicast = true
+		}
+	}
+	if multicast && unicast {
+		return "can not set multicast groupAddress together with unicast ip address", false
+	}
```

Comment:
Will multicast NetworkPolicy support the except field?

Reply:
This is obviously not supported for IGMP rules since the addresses you would specify in the protocol is not a CIDR to begin with. In terms of "dropping udp egress to a multicast CIDR", I would say yes, since the CIDR subtraction logic will be the same for regular and multicast IP addresses